packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/phone.js

```diff
@@ -240,6 +240,69 @@ browserOnly(describe)('plugin-phone', function () {
         }));
     });
 
+    describe('#listActiveCalls()', () => {
+      beforeEach('empty emittedCalls collection', () => {
+        spock.spark.phone.emittedCalls.reset();
+      });
+
+      describe('when no ongoing calls exist', () => {
+        it('should be empty', () =>
+          spock.spark.phone.listActiveCalls()
+            .then((callList) => {
+              assert.isTrue(Array.isArray(callList));
+              assert.equal(callList.length, 0);
+            }));
+      });
+
+      describe('when a ongoing call exists', () => {
+        let connectedCall;
+        beforeEach('create an existing call', () => handleErrorEvent(spock.spark.phone.dial(mccoy.email), (call) => expectCallIncomingEvent(mccoy.spark.phone)
```

Comment:
this might be flaky. I think you need the `expectCallIncomingEvent` down in the `Promise.all()`.

Reply:
This was taken from an existing test, line 268 on `call.js`

Reply:
Should I change that one too to increase stability?

Reply:
Fixed this one for now.